.github/workflows/deploy.sh

```diff
@@ -12,7 +12,7 @@ fi
 
 # cleanup and generate gpg keys
 if [ -d "$HOME/.gnupg" ]; then
-    shred -v ~/.gnupg/*
+    find ~/.gnupg/ -type f -exec shred -v {} \;
```

Comment:
this aligns with what's done already after publishing the package https://github.com/CurrencyFair/OneSignal-Java-SDK/blob/e6d80cd137dc3d26a9cb48692e75ce0919199917/.github/workflows/deploy.sh#L25